### PR TITLE
Enable --engine and --temperature options; fix docs

### PR DESCRIPTION
**Pull request by petrgazarov**
Hey, thanks for maintaining this library.

I added several improvements. Mainly, I wanted to run this lib with `gpt-4` and noticed that the `--engine` option was removed, possibly unintended. I brought it back and also fixed a few issues along the way 😎.

In this PR:
- Add back `--engine` and `--temperature` options and pass them to chat completions API.
- Fix the mode where markdown is turned off. It's broken on main (but works in the latest release).
- Fix docs (clarify usage, options, fix grammar errors here and there)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,87 +44,50 @@ or
...
-   # Add jucasoliveira/terminalGPT as a remote
-   git remote add upstream https://github.com/jucasoliveira/terminalGPT.git
-
-   # Install packages
-   npm install
-   ```
-
-3. On your terminal, type `npm run chat`. Your terminalGPT will start. 😊
-
-<br>
```
Comment (petrgazarov) [inline README.md:125]: Note this entire section is a duplicate and already exists in `CONTRIBUTING.md`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,18 +25,6 @@ Use the steps below:
...
-
-'npx terminalgpt' doesn't install the terminalgpt package, instead it downloads the package to your pc and directly executes it from the cache.
-
-You can find the package using
-
-`ls ~/.npm/_npx/*/node_modules`
-
-To delete the package, you can use
-
-`rm -r ~/.npm/_npx/*/node_modules/terminalgpt`
```
Comment (petrgazarov) [inline CONTRIBUTING.md:40]: This section doesn't belong in `CONTRIBUTING.md` and is also a duplicate from `README.md`.

**Comment by petrgazarov** [comment]
I took a look at the failed GitHub Action, and here is what I found in the [docs](https://github.com/ArtiomTr/jest-coverage-report-action/blob/f1a1382101d9b92b38e3df05c9ed89a834af97c8/README.md?plain=1#L59):

```
HttpError: Resource not accessible by integration
    at /home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:8:323774
    at processTicsAndRejections (node:internal/process/task_queues:96:5)
    at async /home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:64:2535
    at async Ie (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:63:156)
    at async S_ (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:64:2294)
```

> It means that action is running with low privileges. By default, `pull_request` event doesn't have any write permissions, when PR is coming from fork. To fix that, change trigger action to `pull_request_target`:

**Review by lucasgooliveira — APPROVED** [review]
LGTM

**Pull request merged**